index.md

```diff
@@ -24,11 +24,10 @@ When adopting [Kubernetes](https://kubernetes.io), we introduce new risks to our
 * [K08: Secrets Management Failures](./2022/en/src/K08-secrets-management)
 * [K09: Misconfigured Cluster Components](./2022/en/src/K09-misconfigured-cluster-components)
 * [K10: Outdated and Vulnerable Kubernetes Components](./2022/en/src/K10-vulnerable-components)
-* [Other Risks to Consider](./2022/en/other-risks)
-
+* [Other Risks to Consider](./2022/en/src/other-risks)
```

Comment:
```suggestion
* [Other Risks to Consider](./2022/en/src/other-risks.md)
```

Reply:
Similar to K00-K10 it should not require the `.md`